superbench/runner/runner.py

```diff
@@ -173,6 +178,79 @@ def fetch_results(self):    # pragma: no cover
             )
         )
 
+    def __create_results_summary(self):    # pragma: no cover
+        """Create the result summary files."""
+        all_results = list()
+        for node_path in (self._output_path / 'nodes').glob('*'):
+            if not node_path.is_dir():
+                continue
+            results_summary = dict()
+            reduce_ops = dict()
+            file_list = natsorted([str(f) for f in node_path.glob('**/results.json')])
+            file_list = [Path(f) for f in file_list]
+            for results_file in file_list:
+                with results_file.open() as f:
+                    benchmark_name = results_file.parts[-3]
+                    results = json.load(f)
+                    for result in results:
+                        if benchmark_name.endswith('_models'):
+                            benchmark_name = '{}-{}'.format(results_file.parts[-3], result['name'])
+                        if benchmark_name not in results_summary:
+                            results_summary[benchmark_name] = defaultdict(list)
+                        for metric in result['result']:
+                            metric_name = '{}-{}'.format(benchmark_name, metric)
+                            if metric_name not in reduce_ops:
+                                reduce_ops[metric_name] = result['reduce_op'][metric]
+
+                            results_summary[benchmark_name][metric].append(result['result'][metric])
+
+            results_summary = self.__merge_all_metrics(results_summary, reduce_ops)
+            with (node_path / 'results-summary.json').open(mode='w') as f:
+                json.dump(results_summary, f, indent=2)
+
+            results_summary['node'] = node_path.name
+            all_results.append(results_summary)
+
+        with (self._output_path / 'results-summary.jsonl').open(mode='w') as f:
+            for result in all_results:
+                json.dump(result, f)
```

Comment:
how can you differ which result is for which node?

Reply:
Have added one field to indicate the node:
`results_summary['node'] = node_path.name`